_posts/2023-08-16-release-2.6.0.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Release 2.6.1 of MDAnalysis
+---
+
+We are happy to release version 2.6.1 of MDAnalysis!
```

Comment:
```suggestion
We are happy to release version 2.6.1 of MDAnalysis!
(This blog post describes 2.6.1 and the initial release in the 2.6 series, 2.6.0.)
```
Just to make clear that all the changes below really belong to 2.6.0 without going into details (which I don't know) about why we needed 2.6.1 right away... I assume it's numpy and conda-forge...